compiler/src/iree/compiler/Preprocessing/Common/ConvertConvToChannelsLast.cpp

```diff
@@ -30,6 +30,11 @@ namespace mlir::iree_compiler::Preprocessing {
 #define GEN_PASS_DEF_CONVERTCONVTOCHANNELSLASTPASS
 #include "iree/compiler/Preprocessing/Common/Passes.h.inc" // IWYU pragma: export
 
+static llvm::cl::opt<bool>
```

Comment:
I dont think you need this. All preprocessing passes are just stitched together just like you would with `mlir-opt --pass-pipeline="..."` . So adding a pass option is enough.